app/models/suggestion.py

```diff
@@ -0,0 +1,79 @@
+from .. import db
+
+class Suggestion(db.Model):
```

Comment:
We perhaps also need some info about the person who is adding the suggestion (name, email address, phone number, etc.)